src/StackAdmin/Azs.Update.Admin/Module/Azs.Update.Admin/Azs.Update.Admin.psd1

```diff
@@ -17,7 +17,7 @@ Licensed under the MIT License. See License.txt in the project root for license
     RootModule        = 'Azs.Update.Admin.psm1'
 
     # Version number of this module.
-    ModuleVersion     = '0.2.0'
+    ModuleVersion     = '0.2.1'
```

Comment:
@bganapa @deathly809 do we want to update the `ReleaseNotes` field in this file with the same information that was put into the change log for this version?

Reply:
Thanks. Done